docs/publishing/sections/8_ckan.md

```diff
@@ -109,6 +118,15 @@ You can add the `--publish` flag to actually upload artifacts to CKAN after they
 are written to GCS. You must be using a production bucket to publish, either
 by setting `$CALITP_BUCKET__PUBLISH` or using the `--bucket` flag. In addition,
 you may specify a manifest file in GCS if desired.
+
 ```bash
 poetry run python scripts/publish.py publish-exposure california_open_data --bucket gs://calitp-publish --manifest gs://calitp-dbt-artifacts/latest/manifest.json --publish
 ```
+
+## In greater detail: the publish.py script
```

Comment:
can you put this section + specifics of the YAML/exposures and manifest.json at the beginning?

Overall I think it would be useful to organize it with

1. describe CKAN
2. describe publish.py and the exposures yaml, and how they work
3. Making changes to an existing dataset (i.e. non-internal process)
3a. Creating a new dataset

Reply:
Resolved in [4d6b182](https://github.com/cal-itp/data-infra/pull/2839/commits/4d6b182483b5a630658e3f6f2d2ed0d2f5ea1647)